apps/centralised-indexer/testing/CentralisedIndexerTesting.ts

```diff
@@ -0,0 +1,89 @@
+import { Testing, TestingGroup } from '@defichain/jellyfish-testing'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ApiClient } from '@defichain/jellyfish-api-core'
+import { NestFastifyApplication } from '@nestjs/platform-fastify'
+import { CentralisedIndexerStubServer } from './CentralisedIndexerStubServer'
+import { DynamoDbContainer } from './containers/DynamoDbContainer'
+
+/**
+ * OceanApi Testing framework.
```

Comment:
Might want to change this to reflect class.

Reply:
Thanks for catching!